### PR TITLE
Fix: Update tab bar titles on language change

### PR DESCRIPTION
**Pull request by victorsaccucci**
- [x] This pull request is on a [separate branch](https://docs.github.com/en/get-started/quickstart/github-flow) and not the main branch
- [x] I have tested my code with the `xcodebuild` and `swiftlint --strict` commands as directed in the [testing section of the contributing guide](https://github.com/scribe-org/Scribe-iOS/blob/main/CONTRIBUTING.md#testing)

---

### Description

Added logic to dynamically update the "Installation" tab bar item title when the language changes. The solution involved adding the following line to the InstallationViewController:

``` 
title = NSLocalizedString("app.installation.title", value: "Installation", comment: "")
```

arabic

### Observation
I forgot to remove the following code from SettingsViewController, which can be safely removed without any issues:

```
@objc func setText() {
    tabBarController?.tabBar.items?[0].title = NSLocalizedString("Installation", comment: "").uppercased()
}

override func viewWillDisappear(_ animated: Bool) {
    super.viewWillDisappear(animated)
    NotificationCenter.default.removeObserver(self)
}
```

These were added while I was working on solving the tab bar title issue, and they are no longer needed.

- #517

### CODE REVIEW

File: Scribe/SettingsTab/SettingsViewController.swift [hunk 1]
```diff
@@ -82,6 +82,16 @@ final class SettingsViewController: UIViewController {
...
   }
 
+  @objc func setText() {
+      tabBarController?.tabBar.items?[0].title = NSLocalizedString("Installation", comment: "").uppercased()
+  }
+
+  override func viewWillDisappear(_ animated: Bool) {
+      super.viewWillDisappear(animated)
+      removeObserver(object: NotificationCenter.default)
+      NotificationCenter.default.removeObserver(self)
```
Comment (andrewtavis) [inline Scribe/SettingsTab/SettingsViewController.swift:92]: Would you be able to take a look at this warning here, @victorsaccucci? :)

Comment (andrewtavis) [reply Scribe/SettingsTab/SettingsViewController.swift:91]: Note that the `removeObserver(...)` was removed in 023d8e2.

**Comment by andrewtavis** [comment]
Functionality wise this is definitely going in the right direction, @victorsaccucci 😊 Do you have an idea for why we have to switch to the tab in order to get the tab bar label to be localized though? Ideally we'd change the language and everything would be in the given language.

**Comment by victorsaccucci** [comment]
I created this viewWillDisappear method inside the SettingsViewController class, and if you check the original source code before my push, it’s not there. So, since it doesn’t make any difference to the app’s functionality, it can be removed.

Now I pulled from the original repository into my branch and only added the following code:

```
title = NSLocalizedString("app.about.title", value: "Installation", comment: "")
```

**Review by andrewtavis — APPROVED** [review]
What a saga :) Thanks much for the initial work here, @victorsaccucci! The solution here was as follows:
- I removed all the cases where we were setting the title on tab switch, which your contribution turned me on to
- I set them all on view load for the installation tab
- The above ultimately works as we have the app set up to always load the installation tab first, as this is needed to assure that when new keyboards are installed, that the user cannot just navigate back to the settings tab which then would not load and update the available keyboards to changes settings for

Really great to have your support on the project!

**Pull request merged**